python-requirements.txt

```diff
@@ -24,6 +25,7 @@ pygments
 pytest
 pytest-timeout
 pyyaml
+scipy
```

Comment:
... and this. They are dependencies of the chipwhisperer package and installed through it.

Reply:
Done as well. The reason for adding them was ChipWhisperer mentioning that sometimes installation/compilation of these packages fails when using chipwhisperer in editable mode.